### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by gsmet** [comment]
I don’t think that’s a good idea. Maybe a specific task to only generate the docs would be better?

**Comment by geoand** [comment]
@gsmet ideally yes. But at the moment we don't know enough about Azure to know how we can detect doc only changes and then launch a specific job.

This has already bitten us so this is a fix that will at least ensure master doesn't break

File: docs/src/main/asciidoc/extension-authors-guide.adoc [hunk 1]
```diff
@@ -1357,7 +1357,6 @@ The Jackson extension will then use the produced build item to register a module
 If you need more customization capabilities than registering a module,
 you can produce a CDI bean that implements `io.quarkus.jackson.ObjectMapperCustomizer` via an `AdditionalBeanBuildItem`.
 More info about customizing Jackson can be found on the JSON guide link:rest-json-guide.html#configuring-json-support[Configuring JSON support]
-----
```
Comment (machi1990) [inline docs/src/main/asciidoc/extension-authors-guide.adoc]: Ah you beat me to it. I was preparing a PR :-)

**Comment by gsmet** [comment]
Why not just add a job that builds the doc only. 

It would be easy and shouldn’t have a big cost on standard runs.

**Comment by geoand** [comment]
I don't see how that would work in Azure Pipelines. If you have an example, then that would be great.

**Comment by gsmet** [comment]
Ah but the excludes are global

Frankly it’s the first time it happens and we will probably have a lot of doc fixes, so I would just live with it for now to avoid overloading CI in these critical times.

**Comment by geoand** [comment]
I'm fine with whatever, but let's have a decision in chat please soon. Master is currently broken

**Comment by geoand** [comment]
I removed the second commit. Please review and merge. We can decide on whether or not we want CI to run for the docs later

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks.

**Comment by geoand** [comment]
Thanks for the review!

**Pull request merged**